### ISSUE TITLE
Add more detail to surface type questions

### ISSUE THREAD

**Issue by Skivling** [OPENER-next/OpenStop#290]
### Idea

It would be better to have more refined options for 'What is the surface?' questions, rather than only generic paved / unpaved.
The main ones (I think) to include are:
- Asphalt
- Concrete
- Paving stones
- Gravel
- Generic paved
- Generic unpaved

### Bug Relation

No

### Implementation

Maybe the user chooses paved or unpaved first, then it shows more detailed options for that selection. This would reduce the amount of options shown in one screen by splitting them, which could be useful.

### Additional Info

Wiki page: https://wiki.openstreetmap.org/wiki/Key:surface

**Comment by Robbendebiene** [comment]
Hi thanks for the feature request. What you propose is exactly what we planned back then. We just didn't find the time yet 🙂

**Comment by Skivling** [comment]
Of course, focus on the issues and features that are most important first.
